src/components/axes/timeAxis.ts

```diff
@@ -234,12 +247,19 @@ export module Axis {
     public orient(): string;
     public orient(orientation: string): Time;
     public orient(orientation?: string): any {
-      if (orientation && (orientation.toLowerCase() === "right" || orientation.toLowerCase() === "left")) {
-        throw new Error(orientation + " is not a supported orientation for TimeAxis - only horizontal orientations are supported");
+      if (orientation) {
+        Time.verifyTimeAxisOrientation(orientation);
       }
       return super.orient(orientation); // maintains getter-setter functionality
     }
 
+    private static verifyTimeAxisOrientation(orientation: string) {
+      orientation = orientation.toLowerCase();
+      if (["bottom", "top"].indexOf(orientation.toLowerCase()) === -1) {
+        throw new Error(orientation + " is not a supported orientation for TimeAxis - only horizontal orientations are supported");
```

Comment:
"Axis.Time" instead of "TimeAxis". Maybe also change it to "Valid orientations are 'top' and 'bottom' "?